scripts/common.sh

```
@@ -0,0 +1,77 @@
+# Get feature directory path
+get_feature_dir() {
+    local repo_root="$1"
+    local branch="$2"
+    echo "$repo_root/specs/$branch"
+}
+
+# Get all standard paths for a feature
+# Usage: eval $(get_feature_paths)
+# Sets: REPO_ROOT, CURRENT_BRANCH, FEATURE_DIR, FEATURE_SPEC, IMPL_PLAN, TASKS
+get_feature_paths() {
+    local repo_root=$(get_repo_root)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)